iosApp/iosApp/Pages/NearbyTransit/NoNearbyStopsView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  NoNearbyStopsView.swift
+//  iosApp
+//
+//  Created by Horn, Melody on 2024-11-07.
+//  Copyright © 2024 MBTA. All rights reserved.
+//
+
+import SwiftUI
+
+struct NoNearbyStopsView: View {
+    let hideMaps: Bool
+    let onOpenSearch: () -> Void
+    let onPanToDefaultCenter: () -> Void
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 16) {
+            HStack(spacing: 16) {
+                Image(.mbtaLogo)
+                    .resizable()
+                    .accessibilityHidden(true)
+                    .frame(width: 48, height: 48)
+                Text("No nearby stops")
+                    .font(Typography.title2Bold)
+            }
+            .accessibilityAddTraits(.isHeader)
+            .accessibilityHeading(.h2)
+            Text("You’re outside the MBTA service area.")
```

Comment:
This text isn't wrapping in the way that I'd expect it to
![IMG_0198](https://github.com/user-attachments/assets/ac066bff-e383-4ca5-9e48-9f9a794dfb04)

Reply:
That's really weird, because in languages where the first-draft translation of that text is longer, it flows onto two lines correctly, and in the preview, every single piece of text can wrap onto several lines:

|language|preview|
|---|---|
|![Simulator Screenshot - iPhone 13 - 2024-11-15 at 09 49 43](https://github.com/user-attachments/assets/13725ebd-2ec4-463b-9803-1531b3b3544c)|<img alt="Screenshot 2024-11-15 at 10 01 04 AM" src="https://github.com/user-attachments/assets/ef32d031-dc2d-4782-a529-3d011f37576b">|

Reply:
Huh! I do see the full string if I expand the sheet which is unintuitive , maybe a frame issue?

Reply:
Fixed.